pinot-core/src/test/java/org/apache/pinot/util/validator/BasePinotConfigValidatorTest.java

```
@@ -0,0 +1,62 @@
+package org.apache.pinot.util.validator;
+
+import com.github.fge.jsonschema.core.exceptions.ProcessingException;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import org.apache.pinot.core.util.validator.BasePinotConfigValidator;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class BasePinotConfigValidatorTest extends BasePinotConfigValidator {
+
+  @Test
+  public void testLoadingSchema()
+      throws IOException, ProcessingException {
+    // _schema object is null currently
+    Assert.assertNull(_schema);
+
+    loadSchema("/schemas/tableConfig.json");
+    // _schema is store a valid schema after loading
+    Assert.assertNotNull(_schema);
+
+    // fail and throw IOException if schema not found
+    try {
+      loadSchema("/no/schema/here");
+      Assert.fail("schema not found");
```

nit: rename error message to : "should fail for missing schema"